sdk/textanalytics/Azure.AI.TextAnalytics/src/AnalyzeBatchActionsOperation.cs

```diff
@@ -327,7 +327,7 @@ Page<AnalyzeBatchActionsResult> NextPageFunc(string nextLink, int? pageSizeHint)
                 {
                     Response<AnalyzeJobState> jobState = _serviceClient.AnalyzeStatusNextPage(RemoveExtraInformationFromNextLink(nextLink));
 
-                    AnalyzeBatchActionsResult result = Transforms.ConvertToAnalyzeOperationResult(jobState.Value, _idToIndexMap);
+                    AnalyzeBatchActionsResult result = Transforms.ConvertToAnalyzeBatchActionsResult(jobState.Value, _idToIndexMap);
```

Comment:
idk the reason for this change, just want to make sure it makes sense given our move to remove "batch" from the analyze api. can you confirm @maririos ? thanks!

Reply:
In .NET I haven't removed the `batch` from analyze. Will do it after this release.

The main reason for this change was I was cleaning up code in the `Transformation.cs` class and I noticed that both `ConvertToAnalyzeOperationResult` and `ConvertToAnalyzeBatchActionsResult` were doing the same thing. So I deleted the first one and consolidated all behavior into the second one.
Once I make the change to remove `batch` I will rename this function

Reply:
ok sounds fair, thanks!

Reply:
@maririos may I ask why delaying the rename? is it because you do not have enough bandwidth before the release?

Reply:
correct. I want to merge everything today and have a clean build for tomorrow. I will do the rename this week though, so no major concerns there